pipeline/pipeline_tasks/queries/stg_analytics/00_01_04_sba_sfdo_all.sql

```diff
@@ -103,7 +104,8 @@ select
   sba_sfdo.revolver_status,
   api_calls.yelp_rating,
   api_calls.yelp_total_reviews,
-  api_calls.yelp_url
+  api_calls.yelp_url,
+  api_calls.civic_dist
```

Comment:
@makfan64 can this be called `congressional_district`?

Reply:
@VincentLa I saw another field with that name and I couldn't figure how it was used in the time I had. I agree that would be a better game.